components/board/customlinux/setup_test.go

```diff
@@ -0,0 +1,47 @@
+//go:build linux
+
+// Package customlinux implements a board running linux
+package customlinux
+
+import (
+	"testing"
+
+	"go.viam.com/test"
+	"go.viam.com/rdk/components/board"
+)
+
+func TestConfigParse(t *testing.T) {
+	emptyConfigPath := "./data/test_config_empty.json"
+	_, err := parsePinConfig(emptyConfigPath)
+	test.That(t, err, test.ShouldNotBeNil)
+	test.That(t, err.Error(), test.ShouldContainSubstring, `"name" is required`)
```

Comment:
you can also do 
```suggestion
	test.That(t, err, test.ShouldBeError, utils.NewConfigValidationFieldRequiredError("somepath", "name")
```
here

Reply:
but you don't have to - just sharing another pattern.